backend/src/services/authService.ts

```diff
@@ -40,7 +40,16 @@ const saveRefreshToken = async (token: string, userId: string) => {
 
 export const getUserDataFromToken = (data: ITokenData): Promise<ITokenData> => Promise.resolve(data);
 
-export const login = async (user: ITransportedUser): Promise<IAuthUser> => {
+export const removeToken = async (token: any): Promise<any> => {
+  const refreshTokenRepository = getCustomRepository(RefreshTokenRepository);
+  const res = refreshTokenRepository.deleteToken(token);
+  return res;
+};
+
+export const login = async (token: string | null, user: ITransportedUser): Promise<IAuthUser> => {
```

Comment:
It's better to swap parameters, then we won't need to assign null to token (see comment in the authRoute.ts file)
```suggestion
export const login = async (user: ITransportedUser, token?: string): Promise<IAuthUser> => {
```